<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_rename">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>rename</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolTip{
	background: none;
	margin-left: 0px;
	margin-right: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources/res.qrc">:/res/text-editor/rename1.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ledit_rename">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid #242A38;
    border-radius: 15px;
    padding: 0 8px;
    background: #4E596F;
    selection-background-color: #F65964;
	color: white;
}</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Doc name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_share">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>share</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus {
   border: none;
   outline: none;
}

QPushButton {
	background: #242A38;
    border-width: 0px;
    border-radius: 5px;
    border-color: #f76163;
}

QPushButton:hover {
background: #4E596F;
}

QPushButton:pressed {
background: #f76163;
}

QToolTip{
	background: none;
	margin-left: 0px;
	margin-right: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/res.qrc">
       <normaloff>:/res/text-editor/share.png</normaloff>:/res/text-editor/share.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_comments">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>show comments</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus {
   border: none;
   outline: none;
}

QPushButton {
	background: #242A38;
    border-width: 0px;
    border-radius: 5px;
    border-color: #f76163;
}

QPushButton:hover {
background: #4E596F;
}

QPushButton:pressed {
background: #f76163;
}

QToolTip{
	background: none;
	margin-left: 0px;
	margin-right: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/res.qrc">
       <normaloff>:/res/text-editor/comments.png</normaloff>:/res/text-editor/comments.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_connected">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>show connected users</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:focus {
   border: none;
   outline: none;
}

QPushButton {
	background: #242A38;
    border-width: 0px;
    border-radius: 5px;
    border-color: #f76163;
}

QPushButton:hover {
background: #4E596F;
}

QPushButton:pressed {
background: #f76163;
}

QToolTip{
	background: none;
	margin-left: 0px;
	margin-right: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/res.qrc">
       <normaloff>:/res/text-editor/connected.png</normaloff>:/res/text-editor/connected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_logout">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>38</height>
      </size>
     </property>
     <property name="toolTip">
      <string>logout or edit account</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background: #4e596f;
border: 0px solid #242a38;
border-radius: 19px;
}

QPushButton:hover{
padding: -1px 1px 1px -1px;
}

QPushButton:pressed {
padding: 1px -1px -1px 1px;
}

QPushButton:focus {
   border: none;
   outline: none;
}

QToolTip{
	background: none;
	margin-left: 0px;
	margin-right: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources/res.qrc">
       <normaloff>:/res/account.png</normaloff>:/res/account.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
